tests/test_main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.main import is_isogram
+
+
+def test_empty_string():
+    assert is_isogram('')
+
+
+def test_isogram_string():
+    tested_string = 'abc'
+    assert is_isogram(tested_string)
+
+
+def test_non_isogram_string():
+    tested_string = 'abccbc'
+
+    assert not is_isogram(tested_string)
```

Comment:
Add test that check that `is_isogram` is case insencitive (returns False for "Aa")